unity-renderer/Assets/Desktop/Scripts/MainScripts/DCL/Controllers/HUD/SettingsPanelHUDDesktop/Scripts/ScreenResolutionUtils.cs

```diff
@@ -20,7 +42,10 @@ public static void Apply(DisplaySettings displaySettings)
         // Resolution list goes from the smallest to the biggest, our index is inverted for usage reasons so 0 is the biggest resolution available
         private static Resolution GetResolution(DisplaySettings displaySettings)
         {
-            return Resolutions[Resolutions.Count - 1 - displaySettings.resolutionSizeIndex];
+            int resolutionsCount = Resolutions.Count;
```

Comment:
don't see any benefits of having this variable

Reply:
I needed to call `Resolutions` so the DefaultIndex is set before the code below, I'm not sure how to avoid it 🤔